### DISCUSSION TITLE
Remove shared directories for Virtual Machines

### DISCUSSION THREAD

**Discussion by caburmij** [cockpit-project/cockpit-machines#255] [Q&A]
Is there an option to remove shared directories for VMs in the cockpit-machines package? I cannot currently find this option in the cockpit interface, I only have the option to "Add Shared Directory".

If this is not available would it be possible to add this feature?

Thanks

**Comment by KKoukiou** [comment]
We have not implemented that yet. There is a bugzilla reported against RHEL for that https://bugzilla.redhat.com/show_bug.cgi?id=1983848 

I plant to add this feature indeed.
